timemachine/fe/single_topology_v3.py

```diff
@@ -0,0 +1,539 @@
+import warnings
+from collections.abc import Iterable
+
+import numpy as np
+
+from timemachine.fe import system, topology, utils
+from timemachine.fe.dummy import identify_dummy_groups, identify_root_anchors
+from timemachine.lib import potentials
+
+
+class MultipleAnchorWarning(UserWarning):
+    pass
+
+
+class CoreBondChangeWarning(UserWarning):
+    pass
+
+
+class MissingAngleError(RuntimeError):
+    pass
+
+
+def get_romol_bonds(mol):
+    """
+    Return bond idxs given a mol. These are not canonicalized.
+    """
+    bond_list = []
+    for bond in mol.GetBonds():
+        bond_list.append([bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()])
+    return bond_list
```

Comment:
Nit: PR adds two copies of this function -- prefer the one added to `fe/utils.py` https://github.com/proteneer/timemachine/blob/94c67b8cd679a347528cbef2fd03b2256cfeb7da/timemachine/fe/utils.py#L267-L274 ?

Reply:
oops!